_includes/samples/PriceFeeds/PriceConsumerV3.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"log"
+
+	proxy "github.com/aelmanaa/chainlink-price-feed-golang/aggregatorv3"
```

Comment:
Suggestion:
Proxy is an overloaded term and means a different thing in architecture/ system design and in the  client-server world.  Perhaps something a little more descriptive like `AggregatorGen` or something?

Also, i'm not entirely clear on why we are importing this when the previous step wgets it and then abi gens it locally?  cant the local generated file be used instead of importing from github?  May be worth clarifying in the text of the docs.

Reply:
actually "0x9326BFA02ADD2366b30bacB125260Af641031331" is the address of a proxy and not an aggregator (cf. https://docs.chain.link/docs/architecture-decentralized-model/#contracts-overview)

The proxy contract implements the aggregator V3 interface :

1. here you can see that the [AggregatorProxy](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol) implements the [AggregatorProxyInterface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/interfaces/AggregatorProxyInterface.sol)
2.  The AggregatorProxyInterface implements the [AggregatorV2V3Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/interfaces/AggregatorV2V3Interface.sol)
3. Finally the AggregatorV2V3Interface implements [AggregatorV3Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/interfaces/AggregatorV3Interface.sol)

With the same piece of code you can directly call an Aggregator or call it through the proxy. The recommended best practices are to **always** pass by the proxy contract. That's the reason why I don't want to mislead the developers by calling my instance `aggregator`.

However, I'll make some changes: will replace `proxy` by `aggregatorV3` and `instance` by `proxyInstance` . It's very important that people are aware they are calling a proxy

Regarding your second comment, my module name is called "github.com/aelmanaa/chainlink-price-feed-golang" and I've got a local package under aggregaorv3 folder (you can see the structure [here](https://github.com/aelmanaa/chainlink-price-feed-golang)) so when you import local pages ,you have to import "<module_name>/<package_dir_name>")